### PR TITLE
calendar is ready

### PR DESCRIPTION
**Pull request by lytvyshko**
- [DEMO LINK](https://lytvyshko.github.io/layout_calendar/)
- [TEST REPORT LINK](https://lytvyshko.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,72 @@
+$squareSize: 100px;
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: you should create folders with files for each block and sass options, and `@import` them in main.scss

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+html {
+  font-family: Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.page__calendar {
+  width: 726px;
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: I think, it's better to use variables, to explain where did `726px` came from

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,72 @@
...
+  margin: 50px auto 0;
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: in my opinion, you shoul center your calendar with flex property

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,72 @@
...
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  @each $day, $margin in $days {
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: you should replace your loop as well

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,72 @@
...
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+}
+
+.calendar__day {
+  display: flex;
+  width: $squareSize;
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: you may use `mixin` or `placeholder` for the square

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,72 @@
...
+  height: $squareSize;
+  background-color: #eee;
+  border: 1px solid black;
+  font-size: 30px;
```
Comment (alexandra-protyanova) [inline src/styles/main.scss]: it's better to replace into `html`

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/styles/blocks/calendar.scss [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    &--month-length--#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+}
+
+.calendar__day {
+  display: flex;
+  width: $squareSize;
+  height: $squareSize;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/calendar.scss:23]: Use nesting

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! 👍

**Pull request closed without merging**